fern/definition/medical/fhir.yml

```diff
@@ -35,6 +37,16 @@ service:
               formatted `YYYY-MM-DD` as per ISO 8601. If not provided,
                no end date filter will be applied.
             type: optional<string>
+          conversionType:
+            docs: |
+              The doc type to convert to. Either html or pdf.
```

Comment:
nit: is there a way to specify formatting here? ie be able to highlight `html` and `pdf`

Reply:
yah just wrap in backticks